src/NzbDrone.Core/IndexerSearch/AlbumSearchCommand.cs

```diff
@@ -0,0 +1,11 @@
+﻿using NzbDrone.Core.Messaging.Commands;
+
+namespace NzbDrone.Core.IndexerSearch
+{
+    class AlbumSearchCommand : Command
+    {
+        public int AlbumId { get; set; }
```

Comment:
I just copied the code that I had for the `ArtistSearchCommand`, but we can do that instead. I'm not really sure what calls the code and how much difference it would make.